ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml

```diff
@@ -198,3 +199,19 @@
     name: zh_TW.UTF-8
     state: present
   tags: locales
+
+#####################
+# GCC for Debian 10 #
+#####################
+
+- name: Set gcc/g++ 7 to /usr/bin/gcc and /usr/bin/g++ on Debian10
```

Comment:
This makes me nervous as it could overwrite any existing `/usr/bin/gcc` that the user might have due to having another version installed. Can we avoid this and set `CC`/`CXX` variables when required?

If we can't do it another way it needs the `dont_remove_system` tag to allow us to guard against this, or maybe also `adoptopenjdk` so that end users don't run into trouble with it

Reply:
I'll investigate and update the PR As required :)